projects/ngx-translate/core/tests/translate.directive.spec.ts

```diff
@@ -51,7 +53,7 @@ describe('TranslateDirective', () => {
       ],
       declarations: [App]
     });
-    translate = TestBed.inject(TranslateService);
+    translate = TestBed.get(TranslateService);
```

Comment:
I saw that this had been recently changed, but the tests work either way, so I'm unsure why it changed?

If it needs to be `.inject()`, I'll happily put it back.